master/custom/factories.py

```diff
@@ -369,3 +369,17 @@ class Windows64ICCReleaseBuild(Windows64ReleaseBuild):
     buildersuffix = ".nondebug"
     buildFlags = Windows64ReleaseBuild.buildFlags + windows_icc_build_flags
     factory_tags = ["win64", "icc", "nondebug"]
+
+class WindowsArm32Build(WindowsBuild):
+    buildFlags = ["-p", "ARM", "--no-tkinter", "--no-ssl"]
```

Comment:
Didn't we get SSL working?

Reply:
@zooba 
Yes.  But it's not merged yet.
The PR is here: https://github.com/python/cpython/pull/11779